src/sql/workbench/api/browser/mainThreadAria.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SqlMainContext, MainThreadAriaShape } from 'sql/workbench/api/common/sqlExtHost.protocol';
+import { extHostNamedCustomer } from 'vs/workbench/api/common/extHostCustomers';
+import { Disposable } from 'vs/base/common/lifecycle';
+import * as aria from 'vs/base/browser/ui/aria/aria';
+
+
+@extHostNamedCustomer(SqlMainContext.MainThreadAria)
+export class MainThreadAria extends Disposable implements MainThreadAriaShape {
+
+	constructor() {
+		super();
+	}
+
+	$alert(msg: string, disableRepeat?: boolean): void {
+		aria.alert(msg, disableRepeat);
```

Comment:
I'm not sure it makes sense to expose helper methods that don't have anything to do to our product.